Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/encoding/Base45Decoder.kt

```diff
@@ -69,29 +69,36 @@ object Base45Decoder {
     private fun pow(exp: Int) = int256.pow(exp).toLong()
 
     @Throws(IllegalArgumentException::class)
-    fun decode(input: String): ByteArray =
+    fun decode(input: String, mode: Mode = Mode.STRICT): ByteArray =
         input.toByteArray().asSequence().map {
             decoding_charset[it.toInt()].also { index ->
                 if (index < 0) throw IllegalArgumentException("Invalid characters in input.")
             }
         }.chunked(3) { chunk ->
-            if (chunk.size < 2) throw IllegalArgumentException("Invalid input length.")
-            chunk.reversed().toInt(45).toBase(base = 256, count = chunk.size - 1).reversed()
+            if (mode.isStrict() && chunk.size < 2) throw IllegalArgumentException("Invalid input length.")
```

Comment:
Kind of like this (not tested!)

```suggestion
            if (chunk.size < 2) { if (mode.isStrict()) throw IllegalArgumentException("Invalid input length.") else return ByteArray(0) }
```